### ISSUE TITLE
Add speak messages to the editor feature toggles

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#13384]
When toggling one of the Editor features: Top Toolbar, Spotlight Mode, Fullscreen Mode:

<img width="257" alt="screenshot 2019-01-20 at 15 55 37" src="https://user-images.githubusercontent.com/1682452/51441023-94183a80-1ccd-11e9-8daa-a98da7c8e0d4.png">

there's no feedback for screen reader users. Focus is moved back to the menu toggle button and there's no audible message to inform users about what just happened.

Each time a feature gets activated or deactivated, a `speak()` message should be used.
